.github/workflows/deploy_vercel.yaml

```
@@ -29,10 +30,10 @@ jobs:
       - name: Link Vercel project
         run: vercel link --yes --project ${{ github.event.repository.name }} --token=${{ secrets.VERCEL_TOKEN }}
       - name: Set data repository env variable
-        run:  echo "$DATA_REPOSITORY" | vercel env add DATA_REPOSITORY ${{ inputs.environment }} --token ${{ secrets.VERCEL_TOKEN }}
+        run:  echo -n "$DATA_REPOSITORY" | vercel env add DATA_REPOSITORY ${{ inputs.environment }} --token ${{ secrets.VERCEL_TOKEN }}
```

YAMLlint (1.35.1)

[error] too many spaces after colon

(colons)